lib/galaxy/tool_util/data/__init__.py

```diff
@@ -102,9 +103,30 @@ def exists(self, path: str) -> bool:
             return os.path.exists(path)
 
 
+ErrorListT = List[str]
+FileNameInfoT = TypedDict(
+    "FileNameInfoT",
+    {
+        "found": bool,
+        "filename": str,
+        "from_shed_config": bool,
+        "tool_data_path": str,
+        "config_element": Optional[Element],
+        "tool_shed_repository": Any,
+        "errors": ErrorListT,
+    },
+)
+LoadInfoT = Tuple[List[Union[str, Element]], Dict[str, Any]]
+
+
 class ToolDataTable(Dictifiable):
     type_key: str
     data: List[List[str]]
+    empty_field_value: str
+    empty_field_values: Dict[str, str]
+    filenames: Dict[str, FileNameInfoT]
+    _load_info: LoadInfoT
+    _merged_load_info: List[Tuple[Type, LoadInfoT]]
```

Comment:
```suggestion
    _merged_load_info: List[Tuple[Type[ToolDataTable], LoadInfoT]]
```

and can you please use the same type annotation for `self._merged_load_info` on line 173 below?